api/debug/flags.go

```diff
@@ -26,11 +26,12 @@ import (
 	"os"
 	"runtime"
 
+	"github.com/mattn/go-isatty"
```

Comment:
This is very minor, but I think importing github packages together looks better.

Reply:
I reflected it.
Thanks!